app/models/request.js

```diff
@@ -54,6 +45,11 @@ export default Model.extend({
     return eventType === 'pull_request';
   }),
 
+  uniqRawConfigs: computed('raw_configs', function () {
```

Comment:
This could be done with [computed macro](https://api.emberjs.com/ember/3.9/functions/@ember%2Fobject%2Fcomputed/uniqBy)
```js
uniqRawConfigs: uniqBy('raw_configs', 'source'),
```